Zappy_IA_Report/NN/Code/Tutorial7.py

```
@@ -0,0 +1,18 @@
+#python version 3.8
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)